src/app/app.module.ts

```diff
@@ -18,10 +20,11 @@ import { AuthenticationComponent } from './authentication/authentication.compone
         HeaderComponent,
         FooterComponent,
         HomeComponent,
-        AuthenticationComponent
+        AuthenticationComponent,
+        DashboardComponent
     ],
     imports: [BrowserModule, HttpClientModule, FormsModule, AppRoutingModule],
-    providers: [],
+    providers: [AuthGuard,AuthenticationService],
```

Comment:
Hmm both of these should be added here?

Reply:
I think this is some visual bug because on my end it's all good